deploy/0_preconditions.ts

```diff
@@ -0,0 +1,14 @@
+import { Logger } from 'ethers/lib/utils';
```

Comment:
why are `0_preconditions...` and `1_...` prefixed with `{NUMBER}_` but the other ones arent?

Reply:
Unfinished thought.  The script I refactored started with `0_` but that's a holdover from truffle I think.